web_src/js/utils/image.js

```diff
@@ -0,0 +1,34 @@
+import pngChunksExtract from 'png-chunks-extract';
+
+export async function imageInfo(file) {
```

Comment:
Should be called `pngImageInfo` or something like that as it only handles PNGs and no-ops for everything else.